docs/appendices/release-notes/5.2.6.rst

```diff
@@ -1,20 +1,20 @@
-.. _version_5.2.6:
+.. _version_5.2.7:
```

Comment:
Why change here? Filename is still 5.2.6 (and fixed list is intact, so also belong to 5.2.6)